tests/playwright-tests/admin/pages/AdminPage.ts

```diff
@@ -12,9 +16,28 @@ export default class AdminPage {
     this.manageThemesTopicLink = page.locator(
       '//a[text()="manage themes and topics" ]',
     );
+    this.createNewPublicationButton = page.locator(
+      '//a[contains(text(),"Create new publication")]',
+    );
+    this.testThemeDropdown = page.locator(
+      'select[id="publicationsReleases-themeTopic-themeId"]',
+    );
+    this.testTopicDropdown = page.locator(
+      'select[id="publicationsReleases-themeTopic-topicId"]',
+    );
+    this.publicationLink = (text: string) =>
+      page.locator(`//a[text()="${text}"]`);
   }
 
   async clickSignIn() {
     this.signInButton.click();
   }
+
+  async selectTestThemeAndTestTopic() {
+    await this.testThemeDropdown.selectOption({ label: 'Test theme' });
```

Comment:
I guess we could potentially end up with additional Topics underneath Test Theme, so we might need to add something here to make sure that Test Topic is selected as well.  Happy to leave for now though.

Reply:
Thanks for the above comment, I just thought of maintaining uniqueness in the test respecting automation principles. For  now, I wanted to keep this unchanged